src/app/components/applications/SSV/RegisterValidatorHome/components/SelectOperators/components/FirstSquare/components/OperatorDetails/OperatorDetails.tsx

```diff
@@ -15,10 +15,11 @@ type Props = {
   withCopy?: boolean;
   withoutExplorer?: boolean;
   operator: any // ?? IOperator
```

Comment:
What does this comment means? If it should be IOperator - use this type

Reply:
done